<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>-1</y>
      <width>601</width>
      <height>351</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_invest">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <attribute name="title">
      <string>Investments</string>
     </attribute>
     <widget class="QLineEdit" name="leStartBalance">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leRate">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leMonthlyPayment">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leSavingsTime">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Balance</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lePayments">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Annual Rate (%)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Monthly Payment</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Payments (Months)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Saving Time (Years)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbCalculateInvestmentValue">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Calculate</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="gvInvestments">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>421</width>
        <height>301</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_mortage">
     <attribute name="title">
      <string>Mortage</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>623</width>
     <height>22</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <widget class="QMenu" name="menuFinance_Calculator">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>Finance Calculator</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menuFinance_Calculator"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
  </widget>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
  <action name="actionMortage">
   <property name="text">
    <string>Mortage</string>
   </property>
  </action>
  <action name="actionInvestments">
   <property name="text">
    <string>Investments</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>leStartBalance</tabstop>
  <tabstop>leMonthlyPayment</tabstop>
  <tabstop>lePayments</tabstop>
  <tabstop>leSavingsTime</tabstop>
  <tabstop>leRate</tabstop>
  <tabstop>pbCalculateInvestmentValue</tabstop>
  <tabstop>gvInvestments</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
